bash_completion

```diff
@@ -92,5 +92,5 @@ if [[ -n ${ZSH_VERSION-} ]]; then
   autoload -U +X bashcompinit && bashcompinit
 fi
 
-complete -o default -o nospace -F __nvm nvm
+complete -o default -F __nvm nvm
```

Comment:
That's not what the other bash_completions doing, will be special case, which makes no sense, as the previous PR, you can try `curl --compr[tab]` or `git --vers[tab]`, they'll give you the space.